projects/demo/src/modules/components/action/index.ts

```diff
@@ -27,11 +26,11 @@ import {TuiActionExample5} from './examples/5';
         TuiActionExample5,
         TuiActionModule,
     ],
-    templateUrl: './action.template.html',
+    templateUrl: './index.html',
     encapsulation,
     changeDetection,
 })
-export default class ExampleTuiActionComponent {
+export default class PageComponent {
     protected readonly exampleModule = import('./examples/import/import-module.md?raw');
```

Comment:
Would be nice to drop all those imports as well in favor of default names

Reply:
<img width="235" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/06bd78c5-5d41-488e-8b7d-bedbe64be2cc">